process/testdata/tags.txt

```diff
@@ -0,0 +1,18 @@
+datacenter:us1.staging.dog env:staging site:datad0g.com account:staging app:contexts-deduper-cache availability-zone:us-east-1b aws_account:727006795293 aws_autoscaling_groupname:us1-staging-dog-chinook-k8s-ng-asg-69fcd0d8400c ec2_churn:churntest_10pc env:staging iam_profile:chinook/chinook-kube-kubernetes-node-low-trust image:ami-05a0ad73394be70df instance-type:m5.large k8s.io/cluster-autoscaler/enabled:yes k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io/compute k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io/contexts-deduper-cache k8s.io/cluster-autoscaler/node-template/taint/node:contexts-deduper-cache:noschedule kernel:none kubernetes.io/cluster/chinook:owned kubernetes_cluster:chinook name:datadog_contexts-deduper-cache node-role.kubernetes.io/compute node-role.kubernetes.io/contexts-deduper-cache nodegroup:datadog_contexts-deduper-cache region:us-east-1 role:kube-node security-group-name:chinook-kube-node security-group-name:common security_group_name:chinook-kube-node security_group_name:common team:compute team:metrics-intake host:i-0f6c2697b3cc78279
```

Comment:
In the past, I've found it helpful when doing stuff like this to check in an example of the encoded version and writing a test that makes sure we can decode to avoid situations where the encode/decoder break in a way that let's the `x = decode(encode(x))` tests work, but changes the actual encoded format.     I suggest we do the same here.

Reply:
Good idea.